libs/transloco-keys-manager/src/lib/tests/buildTranslationFiles/ts-extraction/service/src/constructor-injection-1.ts

```
@@ -0,0 +1,181 @@
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  Inject,
+  Input,
+  OnDestroy,
+  OnInit,
+  ViewChild,
+} from '@angular/core';
+import { FormBuilder, FormControl, FormGroup } from '@angular/forms';
+import { MatExpansionPanel } from '@angular/material';
```

ESLint

[error] Unable to resolve path to module '`@angular/material`'.

(import/no-unresolved)